erikiado/jp2_online
# Change the field escuela in the Integrante form to plantel. And add a

optional value escuela to all members

#### What's this PR do?
Change the field escuela in the Integrante form to plantel. And add an optional value escuela to all members

#### Where should the reviewer start?
First, he would to check the file familias/models.py, and see that a the new field escuela was added to Integrante. 
Next, view familias/form and review the changes to IntegranteModelForm, on which escuela was changed to plantel.
Finally, you can also view captura/views.py, in which I change escuela to plantel, and the tests.

#### How should this be manually tested?
Enter as a Capturista to a socioeconomical study, and create a new integrante. Write something in the school field, and if the integrante is an alumno, also you must to choose a plantel. 

After you save, edit the integrante to see that also update works.

#### Any background context you want to provide?
This has been added as per request by stakeholders.


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
